frontend/packages/ceph-storage-plugin/src/components/create-storage-system/payloads.ts

```diff
@@ -0,0 +1,22 @@
+import { apiVersionForModel } from '@console/internal/module/k8s';
+import { CEPH_STORAGE_NAMESPACE } from '../../constants';
+import { StorageSystemKind } from '../../types';
+import { ODF_EXTERNAL_PROVIDERS } from '../../odf-external-providers/external-providers';
+import { StorageSystemModel } from '../../models';
+
+export const createSSPayload = (provider: string): StorageSystemKind => {
+  const providerDetails = ODF_EXTERNAL_PROVIDERS.find((p) => p.name === provider);
+  return {
+    apiVersion: apiVersionForModel(StorageSystemModel),
+    kind: StorageSystemModel.kind,
+    metadata: {
+      name: `odf-${provider}-storage-system`,
+      namespace: CEPH_STORAGE_NAMESPACE,
+    },
+    spec: {
+      name: `odf-${provider}-storage-system`,
```

Comment:
I am keeping the name of subsystems to be same as Storage system.
It will help to debug. Also there will be flash system , so it is not always storage cluster